web/components/issues/issue-modal/form.tsx

```diff
@@ -190,6 +190,7 @@ export const IssueFormRoot: FC<IssueFormProps> = observer((props) => {
 
     reset({
       ...defaultValues,
+      ...(isCreateMoreToggleEnabled ? { ...data } : {}),
```

Comment:
Isn't it better to not reset at all if isCreateMoreToggleEnabled is true? just what to understand.